BepInEx.EnableResize/EnableResize.cs

```diff
@@ -34,19 +34,33 @@ public class EnableResize : BaseUnityPlugin
         [DllImport("user32.dll")]
         public static extern int SetWindowLong(IntPtr hWnd, int nIndex, int dwNewLong);
 
+        // Almost the same: https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowlongptra
         private const int GWL_STYLE = -16;
-        private const int WS_THICKFRAME = 0x40000;
+
+        // https://docs.microsoft.com/en-us/windows/win32/winmsg/window-styles
+        private const int WS_CAPTION = 0XC00000;
         private const int WS_MAXIMIZEBOX = 0x10000;
-        private const string GET_CLASS_NAME_MAGIC = "UnityWndClass";
+        private const int WS_MINIMIZEBOX = 0x20000;
+        private const int WS_SYSMENU = 0x80000;
+        private const int WS_THICKFRAME = 0x40000;
 
+        private const string GET_CLASS_NAME_MAGIC = "UnityWndClass"; //How Anon got this???
         private IntPtr WindowHandle = IntPtr.Zero;
-        private bool prev = false;
+
+        private int style = 0;
+        private bool fs = false;
+        private bool prevFS = true;
+        private int res = 0;
+        private int prevRes = 1;
+        private int borderless = 1;
+        private int prevBorderless = 0;
+        private int borderlessMask = WS_CAPTION | WS_MAXIMIZEBOX | WS_MINIMIZEBOX | WS_SYSMENU | WS_THICKFRAME;
+        private WaitForSecondsRealtime oneSecond = new WaitForSecondsRealtime(1f);
 
         internal void Awake()
         {
-            ConfigEnableResize = Config.Bind("Config", "Enable Resize", false, "Whether to allow the game window to be resized. Requires game restart to take effect.");
-            _ConfigEnableResize = ConfigEnableResize.Value;
-            if (!_ConfigEnableResize) return;
+            ConfigEnableResize = Config.Bind("Config", "Enable Resize", true, "Whether to allow the game window to be resized. Requires game restart to take effect.");
```

Comment:
```suggestion
            ConfigEnableResize = Config.Bind("Config", "Enable Resize", false, "Whether to allow the game window to be resized. Requires game restart to take effect.");
```
Since this plugin is often included in bulk with everything else, it's better to have it disabled by default and let the user enable it. or the modpack author can include config with the plugin enabled if it's desirable.

Reply:
Done